src/services/api/apiBaseQuery.ts

```diff
@@ -4,8 +4,8 @@ import { API_URL, FHIR_API_AUTH_TOKEN } from "../../constants";
 
 const baseQuery = fetchBaseQuery({
   baseUrl: API_URL,
+  credentials: "include",
```

Comment:
have you tried without this ? I'm not sure it's necessary after all

Reply:
I have yes, I get 401 error if I don't set this

Reply:
ok :+1: